guide/sections/wis2node.adoc

```diff
@@ -16,9 +22,11 @@ TODO: to be completed
 
 ==== Connecting with Global Services
 
-This section describes the process by which a WIS node is registered with one or more Global Broker and Global Cache components.
+This section describes the process by which a WIS2 node is registered with one or more Global Broker and Global Cache components.
 
-TODO: to be completed
+A WIS2 node shall allow at least one Global Broker and Global Cache to connect to its local broker and web storage. For this, the WIS2 node shall allow the outgoing IP addresses used by the Global Broker and Global Cache to establish connections to broker and web-storage, and allow the Global Cache to subscribe to the topic corresponding to this WIS2 node using the username and password combination "everyone/everyone".
+
+The list of IP addresses used by Global Brokers and Caches can be obtained from the Global Registry. If a WIS2 node implements source IP filtering to grant a Global Broker and Global Cache access, it shall implement changes to its filterting within one month of the changes being published to the Global Registry.
```

Comment:
The list of IP addresses used by Global Brokers and Caches can be obtained from the Global Registry. If a WIS2 node implements source IP filtering to grant Global Brokers and Global Caches access, it shall implement changes to its filtering within one month of the changes being published to the Global Registry.